### ISSUE TITLE
Erroneous error message "FileWorker ADDED - calling deleteChunks for ..."

### ISSUE THREAD

**Issue by Sojourneer** [CollectionFS/Meteor-CollectionFS#898]
I am getting the above message for every file in db.cfs.sheets.filerecord. I believe these are spurious.

Looking at cfs_worker.js getDoneQuery(), the query does not include the "chunks" condition which is inconsistent with the documentation. I checked the query generated by getDoneQuery() in mongoDB console, and verify it selects all records.

**Comment by travisdahl** [comment]
Yes I updated my packages and got `FileWorker ADDED - calling deleteChunks for xxxx` for every image in my production DB which deleted every image in my production S3. All of my images are gone. Can someone please tell me where this delete gets called so I can make sure it doesnt happen again?
